universal-application-tool-0.0.1/app/auth/AdfsProfileAdapter.java

```diff
@@ -0,0 +1,32 @@
+package auth;
+
+import org.pac4j.oidc.client.OidcClient;
+import org.pac4j.oidc.config.OidcConfiguration;
+import org.pac4j.oidc.profile.OidcProfile;
+
+/**
+ * This class takes an existing UAT profile and augments it with the information from an AD profile.
+ * Right now this is only extracting the email address, as a proof that this works - it needs to be
+ * built out. TODO(nathan): decide what can be extracted and extract it.
+ */
+public class AdfsProfileAdapter extends UatProfileAdapter {
+
+  public AdfsProfileAdapter(
```

Comment:
question: (for this constructor and the other adapter constructor) - should these be `@Inject`? Or do we not want dependency injection here?

Reply:
Thanks for clarifying!